atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/StartupIndependenceMultipleCassandraEteTest.java

```diff
@@ -61,175 +44,43 @@
-        try {
-            addTodo();
-            return true;
-        } catch (AtlasDbRemoteException e) {
-            return false;
-        } catch (Exception e) {
-            if (exceptionIsRetryableAndContainsMessage(e, "is not initialized yet")) {
-                return false;
-            }
-            throw e;
-        }
-    }
-
-    private void assertNotInitializedExceptionIsThrownAndMappedCorrectly() {
-        try {
-            addTodo();
-            fail("Expected to throw an exception");
-        } catch (Exception e) {
-            assertTrue(exceptionIsRetryableAndContainsMessage(e, "CassandraKeyValueService is not initialized yet"));
-        }
-    }
-
-    private static boolean exceptionIsRetryableAndContainsMessage(Exception exc, String message) {
-        // We shade Feign, so we can't rely on our client's RetryableException exactly matching ours.
-        return exc.getClass().getName().contains("RetryableException") && exc.getMessage().contains(message);
-    }
-
-    private static void addTodo() {
-        TodoResource todos = EteSetup.createClientToSingleNode(TodoResource.class);
-        Todo todo = getUniqueTodo();
-
-        todos.addTodo(todo);
-    }
-
-    private static Todo getUniqueTodo() {
-        return ImmutableTodo.of("some unique TODO item with UUID=" + UUID.randomUUID());
-    }
-
-    private static void runOnCassandraNodes(List<String> nodes, CassandraContainerOperator operator)
-            throws InterruptedException {
-        ExecutorService executorService = Executors.newFixedThreadPool(nodes.size());
-
-        executorService.invokeAll(nodes.stream()
-                .map(cassandraContainer -> Executors.callable(() -> operator.nodeOperation(cassandraContainer)))
-                .collect(Collectors.toList()));
-    }
-
-    private interface CassandraContainerOperator {
-        void nodeOperation(String node);
-    }
-
-    public static void killCassandraContainer(String containerName) {
-        Container container = EteSetup.getContainer(containerName);
-        try {
-            container.kill();
-        } catch (IOException | InterruptedException e) {
-            throw Throwables.propagate(e);
-        }
-    }
-
-    public static void startCassandraContainer(String containerName) {
-        Container container = EteSetup.getContainer(containerName);
-        try {
-            container.start();
-        } catch (IOException | InterruptedException e) {
-            throw Throwables.propagate(e);
-        }
-        waitForCassandraContainer(container);
-    }
-
-    private static void waitForCassandraContainer(Container container) {
-        DockerPort containerPort = new DockerPort(container.getContainerName(), CASSANDRA_PORT, CASSANDRA_PORT);
-        Awaitility.await()
-                .atMost(60, TimeUnit.SECONDS)
-                .until(containerPort::isListeningNow);
+        StartupIndependenceUtils.restartAtlasWithChecks();
```

Comment:
There's some feature envy here - consider moving `verifyCassandraIsSettled()` into the Utils class, too.